clang/include/clang-c/Dependencies.h

```diff
@@ -380,6 +380,210 @@ clang_experimental_DependencyScannerWorker_getFileDependencies_v5(
     CXModuleLookupOutputCallback *MLO, unsigned Options,
     CXFileDependenciesList **Out, CXDiagnosticSet *OutDiags);
 
+/**
+ * Output of \c clang_experimental_DependencyScannerWorker_getDependenciesGraph.
+ */
+typedef struct CXOpaqueDependenciesGraph *CXDependenciesGraph;
```

Comment:
Maybe "DependencyGraph"? I'm not used to seeing it pluralized like this.  Same for the method `clang_experimental_DependencyScannerWorker_getDependenciesGraph`.

Reply:
Renamed `CXDependenciesGraph` -> `CXDepGraph`.